example/spec/parallel/SyncGranuleTask/ChecksumOneGigFileSpec.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+const get = require('lodash/get');
+const pAll = require('p-all');
+const pTimeout = require('p-timeout');
+const { pullStepFunctionEvent } = require('@cumulus/message/StepFunctions');
+const { randomId } = require('@cumulus/common/test-utils');
+
+const { createCollection } = require('@cumulus/integration-tests/Collections');
+const { createProvider } = require('@cumulus/integration-tests/Providers');
+
+const { deleteCollection } = require('@cumulus/api-client/collections');
+const { deleteProvider } = require('@cumulus/api-client/providers');
+const { lambda } = require('@cumulus/aws-client/services');
+const S3 = require('@cumulus/aws-client/S3');
+
+const { loadConfig } = require('../../helpers/testUtils');
+const { fetchFakeS3ProviderBucket } = require('../../helpers/Providers');
+
+describe('The SyncGranule task with a 1 GB file to be checksummed', () => {
+  let beforeAllFailed = false;
+  let collection;
+  let granuleId;
+  let prefix;
+  let provider;
+  let sourceBucket;
+  let syncGranuleOutput;
+
+  beforeAll(async () => {
+    try {
+      const config = await loadConfig();
+
+      prefix = config.stackName;
+      sourceBucket = await fetchFakeS3ProviderBucket();
+
+      // Create the collection
+      collection = await createCollection(
+        prefix,
+        {
+          duplicateHandling: 'version',
+          process: 'modis',
+        }
+      );
+
+      // Create the S3 provider
+      provider = await createProvider(prefix, { host: sourceBucket });
+
+      granuleId = randomId('granule-id-');
+
+      const FunctionName = `${prefix}-SyncGranule`;
+
+      const functionConfig = await lambda().getFunctionConfiguration({
+        FunctionName,
+      }).promise();
+
+      const Payload = JSON.stringify({
+        cma: {
```

Comment:
Not for this PR, but this feels like the sort of thing it'd be nice to boilerplate and store with the component,  particularly if we start testing individual lambdas (which we should be).